.github/workflows/conda-python-tests.yaml

```diff
@@ -73,21 +73,21 @@ jobs:
               # amd64
               - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.4.3', LINUX_VER: 'centos7',     GPU: 'v100', DRIVER: 'earliest' }
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
-              - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
+              - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
               # arm64
               - { ARCH: 'arm64', PY_VER: '3.10', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu22.04', GPU: 'a100', DRIVER: 'latest'   }
             nightly:
               # amd64
               - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.4.3', LINUX_VER: 'centos7',     GPU: 'v100', DRIVER: 'earliest' }
               - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.4.3', LINUX_VER: 'ubuntu20.04', GPU: 'v100', DRIVER: 'latest'   }
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'rockylinux8', GPU: 'v100', DRIVER: 'latest'   }
-              - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
               - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
+              - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '12.0.1', LINUX_VER: 'ubuntu22.04', GPU: 'v100', DRIVER: 'latest'   }
```

Comment:
Promise I didn't ignore your previous comment asking for that combination.

On the current state of this branch, that exact combination is being tested *on PRs* for `conda-python-tests` workflow, which is why I didn't also add it to the nightly matrix.

Does that change you opinion on this suggestion?

Or do you think it's desirable to have that combination in both places because we don't enforce that PRs have to be up to date with the target branch before merging?

Reply:
Just pushed https://github.com/rapidsai/shared-workflows/pull/176/commits/115ad5aa03616c5e3464da7e65b0efd113059871. As of that commit, I believe this branch is meeting these constraints for test job matrices:

* at least 1 PR job for every Python version in `(3.9, 3.10, 3.11)`
* at least 1 PR job for every CUDA version in `(11.8.0, 12.0.1, 12.2.2)`
* at least 1 PR job for every architecture in `(amd64, arm64)`
* am64 + latest CUDA + latest Python + latest Ubuntu present in both PR and nightly
* nightly is a strict superset of what's tested on PRs
* no net change to the number of test jobs on PRs or nightly, relative to the current state of `branch-24.04`
* as-close-as-possible-to-equal spread of test jobs across Python versions

Since that commit only changed `CUDA_VER` and `LINUX_VER`, the table from https://github.com/rapidsai/shared-workflows/pull/176#discussion_r1499601279 is unchanged.